### DISCUSSION TITLE
Remove input scaling in centered instance configuration

### DISCUSSION THREAD

**Discussion by Marielamend** [talmolab/sleap#1716] [Ideas]
Bug related to mismatching crop sizes when doing inference. Led to error during centered instance model training when users set input scaling to 1.25.

**Comment by gitttt-1234** [comment]
Closing this discussion, the input scaling for centered instance is fixed with the [`sleap-nn`](https://github.com/talmolab/sleap-nn) backend, which is the default neural network backend for SLEAP >= v1.5! We now apply [scaling to the crops](https://github.com/talmolab/sleap-nn/pull/381) rather than applying to whole images and then cropping on the scaled image!

Related issues: https://github.com/talmolab/sleap/issues/1993
